libs/klv/src/main/java/org/codice/alliance/libs/klv/Utilities.java

```diff
@@ -25,4 +30,30 @@ public static boolean isEmptyString(Serializable serializable) {
     public static boolean isNotEmptyString(Serializable serializable) {
         return !isEmptyString(serializable);
     }
+
+    static void safelySetAttribute(Metacard metacard, Attribute attribute) {
+        safelySetAttribute(metacard, attribute.getName(), attribute.getValues());
+    }
+
+    static void safelySetAttribute(Metacard metacard, String attributeName, Serializable value) {
+        if (metacard.getMetacardType()
+                .getAttributeDescriptor(attributeName)
+                .getType()
```

Comment:
`getType()` won't be a problem but `getAttributeDescriptor()` might return a null, depending on how this is invoked. It might never happen if your callers are all *correct*, but you shouldn't count on that.